dm/worker/server.go

```diff
@@ -435,8 +447,7 @@ func (s *Server) QueryStatus(ctx context.Context, req *pb.QueryStatusRequest) (*
 	}
 
 	resp.SubTaskStatus = w.QueryStatus(req.Name)
-	resp.RelayStatus = relayStatus
-	resp.Source = w.cfg.SourceID
+	sourceStatus.RelayStatus = relayStatus
```

Comment:
How about setting `RelayStatus` to `nil` to make the response of `query-status` clearer when `enable-relay: false`? cc @WangXiangUSTC .

Reply:
I think it's ok